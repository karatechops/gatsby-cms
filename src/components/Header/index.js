import React from 'react';
import { Link } from 'gatsby';
import { Box, Text } from 'grommet';

export const Header = () => (
  <Box
    direction="row"
    background="dark-1"
    gap="small"
    pad={{ vertical: 'small', horizontal: 'medium' }}
    justify="between"
  >
    <Link to="/">
      <Text weight="bold" color="accent-1">
        Home
      </Text>
    </Link>
    <Link to="/blog">
      <Text weight="bold" color="accent-1">
        Blog
      </Text>
    </Link>
  </Box>
);

export default Header;
