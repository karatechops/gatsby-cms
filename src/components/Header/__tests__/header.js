import React from 'react';
import renderer from 'react-test-renderer';
import { useStaticQuery } from 'gatsby';
import Header from '../index';

beforeEach(() => {
  useStaticQuery.mockReturnValue({
    site: {
      siteMetadata: {
        title: 'My site',
      },
    },
  });
});

describe('Header', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
