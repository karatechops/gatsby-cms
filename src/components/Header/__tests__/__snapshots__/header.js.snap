// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div
  className="StyledBox-sc-13pk1d4-0 dOfRrV"
>
  <a
    href="/"
  >
    <span
      className="StyledText-sc-1sadyjn-0 dPfkAU"
    >
      Home
    </span>
  </a>
  <div
    className="StyledBox__StyledBoxGap-sc-13pk1d4-1 iChEkS"
  />
  <a
    href="/blog"
  >
    <span
      className="StyledText-sc-1sadyjn-0 dPfkAU"
    >
      Blog
    </span>
  </a>
</div>
`;
